migrations/20240321211311_event.up.sql

```diff
@@ -0,0 +1,18 @@
+-- Add up migration script here
+
+CREATE TABLE IF NOT EXISTS event (
+    id BLOB NOT NULL, -- network_id sort_value controller StreamID height event_cid
+    cid BLOB UNIQUE, -- NOT NULL? the cid of the event as bytes no 0x00 prefix
+    ahash_0 INTEGER NOT NULL, -- the ahash is decomposed as [u32; 8]
+    ahash_1 INTEGER NOT NULL,
+    ahash_2 INTEGER NOT NULL,
+    ahash_3 INTEGER NOT NULL,
+    ahash_4 INTEGER NOT NULL,
+    ahash_5 INTEGER NOT NULL,
+    ahash_6 INTEGER NOT NULL,
+    ahash_7 INTEGER NOT NULL,
+    value_retrieved BOOL NOT NULL DEFAULT FALSE, -- bool with 0=false, 1=true. indicates if we have the value
```

Comment:
do we want to use `STRICT` tables? It removes bool and requires using integer and is generally more annoying. It's not recommended by the sqlite docs but could make a sqlite-> postgres migration easier. Shouldn't really be necessary since the rust data should be strongly typed so I left it off.